index.js

```diff
@@ -142,7 +201,16 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
```

Comment:
I think in this function we had the right idea but maybe didn't account for the data type of `movies[I].boxOffice`  - hint: its not a number.